fe/fe-core/src/main/java/com/starrocks/master/ReportHandler.java

```diff
@@ -372,6 +387,18 @@ private static void diskReport(long backendId, Map<String, TDisk> backendDisks)
                 backendId, (System.currentTimeMillis() - start));
     }
 
+    private static void workgroupReport(long backendId, List<TWorkGroup> workGroups) {
+        LOG.info("begin to handle workgroup report from backend{}", backendId);
+        long start = System.currentTimeMillis();
+        Backend backend = Catalog.getCurrentSystemInfo().getBackend(backendId);
+        if (backend == null) {
+            LOG.warn("backend does't exist. id: " + backendId);
```

Comment:
The check only print log?

Reply:
According to diskReport method,  it is a defensive programming style.